src/test/resources/uk/org/okapibarcode/output/code93-with-rotation-270.eps

```diff
@@ -0,0 +1,77 @@
+%!PS-Adobe-3.0 EPSF-3.0
```

Comment:
This EPS file does not display correctly. What OS are you using? On Ubuntu the standard document viewer allows you to open EPS files easily. If you're on Windows, you should be able to find a few online viewers via Google.

Reply:
yeah the EPS one honestly I am not sure HOW to rotate it.  The postscript code looks like hyroglyphics I just googled and found that but its clearly not right.

Reply:
The "BoundingBox" setting needs a swapped rotated width and height, like the Java2D renderer. I think it's more than that, but that's the first thing.

Reply:
Bounding box height and width fixed!